goss.yaml

```diff
@@ -0,0 +1,22 @@
+process:
+  otelcol:
+    running: true
+
+user:
+  _daemon_:
+    exists: true
+    uid: 584792
+    groups:
+    - _daemon_
```

Comment:
This seems brittle.
It would make sense to keep this assertion if the _daemon_ user is some user facing functional promise, but iiuc the name and uid don't matter outside the container..?

I would however add an assertion for a port being bound.